entc/gen/template/config.tmpl

```diff
@@ -24,6 +24,10 @@ type config struct {
 	log func(...interface{})
 	// hooks to execute on mutations.
 	hooks *hooks
+	{{- range $.Nodes }}
+		// optional schema name for the {{ .Name }} table.
+		{{ .Name }}Schema string
+	{{- end }}
```

Comment:
3 comments about this - 
1. I think this should be storage-specific configuration (should move to `dialect/sql/config.tmpl`)
2. I'm not sure this should be added by default to the codegen. There are 2 options: 1) provide a feature-flag to enable it to all schemas 2) add a new annotation to `entsql` that lets users to enable it to specific schemas (shared mixin configuration for all schemas)
3. Struct fields should be private

WDYT?